ait/core/cmd.py

```diff
@@ -577,13 +579,22 @@ def YAMLCtor_include(loader, node):
     name = os.path.join(os.path.dirname(loader.name), node.value)
     data = None
     with open(name,'r') as f:
-        data = yaml.load(f, Loader=yaml.Loader)
+        data = yaml.load(f)
+    return data
+
+
+def YAMLCtor_include(loader, node):  # noqa
+    # Get the path out of the yaml file
+    name = os.path.join(os.path.dirname(loader.name), node.value)
+    data = None
+    with open(name, "r") as f:
+        data = yaml.load(f)
     return data
```

Comment:
This overwrites previous definition of `YAMLCtor_include`. Is this done on purpose?